src/main/java/io/xstefank/wildlfy/bot/format/checks/CommitsQuantityCheck.java

```diff
@@ -0,0 +1,65 @@
+package io.xstefank.wildlfy.bot.format.checks;
+
+import io.xstefank.wildlfy.bot.config.CommitsQuantity;
+import org.kohsuke.github.GHPullRequest;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CommitsQuantityCheck implements Check {
+    private final Pattern VALUE_REGEX = Pattern.compile("^([1-9]|[1-9][0-9]|100)$");
+    private final Pattern RANGE_REGEX = Pattern.compile("^([1-9]|[1-9][0-9]|100)-([1-9]|[1-9][0-9]|100)$");
+    static final String DEFAULT_MESSAGE = "Invalid title content";
+
+    private Integer parsedRangeBottomBoundary;
+    private Integer parsedRangeUpperBoundary;
+    private String message;
+
+    public CommitsQuantityCheck(CommitsQuantity commitsQuantity) {
+        if (commitsQuantity == null || commitsQuantity.quantity == null) {
+            throw new IllegalArgumentException("Quantity was not set");
```

Comment:
so this is a no. We should make all checks optional. So if user doesn't specify the configuration, check isn't included. Not that the app fails.